plugins/docker/docker_

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+#
+# Plugin to graph docker container and storage.
```

Comment:
Please add a perldoc-style documentation to this plugin. Maybe take a look at `plugins/munin/munin_events` for an example.